server/views/_partials/__header.njk

```diff
@@ -20,10 +20,10 @@
         <nav class="content">
           <ul id="navigation">
             <li id="user-bar-email">
-              demo@ch.gov.uk
+              {{ user_profile.email }}
             </li>
             <li id="user-bar-sign-out">
-              <a id="user-signout" href="{{ devHubUrl }}/signout">Sign out</a>
+              <a id="user-signout" href="{{ chsUrl }}/signout">Sign out</a>
```

Comment:
I think this should be `devHubUrl` as we are on a subdomain (dev.etc) so we expect the app to redirect to our implementation of /signout rather than chs.

Reply:
Oh are you sure? We use the chs account for signing in. So we can use their logout endpoint to logout. If we implement our own logout, we'll have to manually call the chs account logout. Is that what's expected?

Reply:
As discussed, the chsUrl will work for now. However there is no option to specify a redirect url (like sign in), so it will go back to the original page you were on. https://github.com/companieshouse/CH-MojoX-SignIn/blob/develop/lib/CH/MojoX/SignIn/Controller/OAuth2.pm#L72